run.NGD_v3atm.F1850.sh

```diff
@@ -0,0 +1,659 @@
+ fincl2 = 'FLUT','PRECT','U200','V200','U850','V850','Z500','OMEGA500','UBOT','VBOT','TREFHT','TREFHTMN:M','TREFHTMX:X','QREFHT','TS','PS','TMQ','TUQ','TVQ','TOZ', 'FLDS', 'FLNS', 'FSDS', 'FSNS', 'SHFLX', 'LHFLX', 'TGCLDCWP', 'TGCLDIWP', 'TGCLDLWP', 'CLDTOT', 'T250', 'T200', 'T150', 'T100', 'T050', 'T025', 'T010', 'T005', 'T002', 'T001', 'TTOP', 'U250', 'U150', 'U100', 'U050', 'U025', 'U010', 'U005', 'U002', 'U001', 'UTOP', 'FSNT', 'FLNT'
+ fincl3 = 'PSL','T200','T500','U850','V850','UBOT','VBOT','TREFHT', 'Z700', 'TBOT:M'
+ fincl4 = 'FLUT','U200','U850','PRECT','OMEGA500'
+ fincl5 = 'PRECT','PRECC','TUQ','TVQ','QFLX','SHFLX','U90M','V90M'
+ fincl6 = 'CLDTOT_ISCCP','MEANCLDALB_ISCCP','MEANTAU_ISCCP','MEANPTOP_ISCCP','MEANTB_ISCCP','CLDTOT_CAL','CLDTOT_CAL_LIQ','CLDTOT_CAL_ICE','CLDTOT_CAL_UN','CLDHGH_CAL','CLDHGH_CAL_LIQ','CLDHGH_CAL_ICE','CLDHGH_CAL_UN','CLDMED_CAL','CLDMED_CAL_LIQ','CLDMED_CAL_ICE','CLDMED_CAL_UN','CLDLOW_CAL','CLDLOW_CAL_LIQ','CLDLOW_CAL_ICE','CLDLOW_CAL_UN'
+ fincl7 = 'O3', 'PS', 'TROP_P'
+
+ tropopause_e90_thrd    = 80.0e-9
+ history_gaschmbudget_2D = .false.
+ history_gaschmbudget_2D_levels = .false.
+ history_UCIgaschmbudget_2D = .false.
+ history_UCIgaschmbudget_2D_levels = .false.
+ gaschmbudget_2D_L1_s =  1
+ gaschmbudget_2D_L1_e = 26
+ gaschmbudget_2D_L2_s = 27
+ gaschmbudget_2D_L2_e = 38
+ gaschmbudget_2D_L3_s = 39
+ gaschmbudget_2D_L3_e = 58
+ gaschmbudget_2D_L4_s = 59
+ gaschmbudget_2D_L4_e = 72
+! history_aero_optics    = .true.
+! history_aerosol        = .true.
+! history_amwg           = .true.
+! history_budget         = .true.
+! history_verbose        = .true.
+ cosp_lite = .true.
+ zmconv_microp = .true.
+ zmconv_clos_dyn_adj = .true.
+ zmconv_MCSP_heat_coeff = 0.3
+ zmconv_tpert_fix = .true.
+ zmconv_ke = 2.5e-6
+ p3_wbf_coeff = 1.0
+
+ sscav_tuning = .false.
+ 
+ !LWCF tuning
+ nucleate_ice_subgrid   =  1.35
+ p3_mincdnc             =  20.D6
+ dust_emis_fact         =  11.8D0
+ seasalt_emis_scale     =  0.55D0
+
+ tracer_cnst_cycle_yr           = 1995
+ tracer_cnst_datapath           = '\$DIN_LOC_ROOT/atm/cam/chem/methane/'
+ tracer_cnst_file               = 'ch4_oxid_1.9x2.5_L26_1990-1999clim.c090804.nc'
+ tracer_cnst_filelist           = ''
+ tracer_cnst_specifier          = 'CH4','cnst_NO3:NO3', 'cnst_OH:OH'
+ tracer_cnst_type               = 'CYCLICAL'
+
+ linoz_psc_t = 198.0
+
+ ! empty sad_file would have no effect for the 4th full smoke config
+
+ sad_file               = '\$DIN_LOC_ROOT/atm/waccm/sulf/SAD_SULF_1849-2100_1.9x2.5_c090817.nc'
+ sad_type     = 'SERIAL' 
+
+ drydep_list = 'O3','H2O2','CH2O','CH3OOH','NO','NO2','HNO3','HO2NO2','PAN','CO','CH3COCH3','C2H5OOH','CH3CHO','H2SO4','SO2','NO3','N2O5'
+
+ gas_wetdep_list                = 'C2H5OOH','CH2O','CH3CHO','CH3OOH','H2O2','H2SO4','HNO3','HO2NO2','SO2','SOAG0','SOAG15','SOAG24','SOAG35','SOAG34','SOAG33','SOAG32','SOAG31'
+ gas_wetdep_method              = 'NEU'
+ 
```

Comment:
@wlin7 @mingxuanwupnnl As I remember, the prescribed volcanic aerosol will not be turned on if MAM5 is turned on, right? So, we may not need that part in the compset.xml